contrib/vms/README.md

```diff
@@ -0,0 +1,70 @@
+# Vagrant development boxes
+
+This directory contains a `Vagrantfile` which can be used to automatically
+create virtual machines for testing purposes. The virtual machines are
+automatically provisioned with all required dependencies for building & testing
+of exiv2 (the provisioning is shared with the GitLab CI).
+
+The following Linux distributions are provided (the name in the brackets is the
+name of the Vagrant VM):
+- Fedora 28 ("Fedora")
+- Debian 9 aka Stretch ("Debian")
+- Archlinux ("Archlinux")
+- Ubuntu 16.04 aka Bionic Beaver ("Ubuntu")
+- CentOS 7 ("CentOS")
+- OpenSUSE Tumbleweed ("OpenSUSE")
+
+The Fedora, Archlinux and OpenSUSE boxes are the 'vanilla' distribution with
+some additional packages installed.
+
+For Debian and Ubuntu, we build gtest manually from source and install the
+resulting library to /usr/lib/.
+
+On CentOS, we have to install a `cmake3` and `python36` (the default cmake is
+too old and a default python3 does not exist) which we symlink to
+`/usr/bin/cmake` & `/usr/bin/python3` to retain a similar workflow to the other
+distributions.
+
+For further details, consult the shell scripts `setup.sh` and
+`ci/install_dependencies.sh`.
+
+
+All boxes come with `conan` installed via pip in the `vagrant` user's home
+directory and the `exiv2` git repository cloned.
+
+
+Please note that these VMs are not continuously tested and the provisioning can
+break. Please open an issue on GitHub if you happen to encounter a problem.
+
+
+## Usage
+
+Please install [Vagrant](https://www.vagrantup.com/) and a supported provider
+(e.g. libvirt, VirtualBox).
+
+Choose a box from the above list and run:
```

Comment:
[suggestion] I think it would be good to indicate that the user has to run this command from the directory where the Vagrantfile is.

Reply:
That's a good idea. I always do that automatically, so I just forgot.